site/docs/v1/tech/apis/_form-field-response-body.adoc

```diff
@@ -0,0 +1,141 @@
+==== {response_body_title}
+
+[.api]
+
+[field]#field.confirm# [type]#[Boolean]#::
+Determines if the `control` field should be verified. If `true`, a confirmation field is included.
+
+[field]#field.consentId# [type]#[UUID]#::
+The Id of an existing link:/docs/v1/tech/apis/consent[Consent].
+
+[field]#field.control# [type]#[String]#::
```

Comment:
These larger sections may need to go in an include so we can re-use them more easily.